R/getAliases.R

```diff
@@ -0,0 +1,28 @@
+
+# get installed packages, making sure that high priority packages are on top
+ip <- installed.packages()
+
+ord <- order(ip[, "Priority"])
+ip <- ip[ord, ]
+
+ret <- list()
+
+for (row in rownames(ip)) {
```

Comment:
Better use `lapply()` instead of `for` in this case to avoid many re-allocation of `ret`:

```r
ret <- lapply(rownames(ip), function(row) {
    libPath <- ip[row, "LibPath"]
    pkg <- ip[row, "Package"]
    filename <- file.path(libPath, pkg, "help", "aliases.rds")
    info <- list(
        package = pkg,
        libPath = libPath,
        aliasFile = filename,
        aliases = NULL
    )
    if (file.exists(filename)) {
        info[["aliases"]] <- as.list(readRDS(filename))
    }
    info
})
names(ret) <- rownames(ip)
```